overlord/devicestate/handlers_install.go

```diff
@@ -250,6 +250,28 @@ func writeTimings(st *state.State, rootdir, fromMode string) error {
 	return nil
 }
 
+// buildInstallObservers will create observers for the gadget assets used
```

Comment:
this is a bit confusing because a single observer is at most created.  This should have a singular name buildInstallObserver.

// buildInstallObserver creates an observer for gadget assets if applicable, otherwise the returned gadget.ContentObserver is nil.
// The observer if any is also returned as non-nil trustedObserver if encryption is  in use.

Reply:
Changed